dist/images/ovn-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# run the ovs-vswitchd daemon from a container
+
+#docker run --rm --network host --mount type=bind,src=/var/run/openvswitch,dst=/var/run/openvswitch  phil-ovn &
+#docker run --rm --network host -v /var/run/openvswitch:/var/run/openvswitch:rw  phil-ovn
+#docker run --rm --network host -v /var/run/openvswitch/db.sock:/var/run/openvswitch/db.sock:ro  phil-ovn
+#docker run --rm --network host -v /var/run/openvswitch/db.sock:/var/run/openvswitch/db.sock:ro -v /etc/origin/node/client-ca.crt:/etc/origin/node/client-ca.crt:ro phil-ovn
+docker run --rm --network host \
+	-e OPENSHIFT_DNS_DOMAIN=cluster.local \
```

Comment:
Just remove the OPENSHIFT_DNS_DOMAIN line, I think.

Reply:
OK